### PR TITLE
Added solution Matrix hw-2.2

### PR DESCRIPTION
**Pull request by nevskyy**

### CODE REVIEW

File: src/main/java/core/basesyntax/MatrixOperation.java [hunk 1]
```diff
@@ -1,23 +1,53 @@
 package core.basesyntax;
 
+//import java.util.Arrays;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MatrixOperation.java]: please remove commented code everywhere

File: src/main/java/core/basesyntax/MatrixOperation.java [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
+            diagonalElts[i] = matrix[i][i];
+        }
+        return diagonalElts;
     }
 
     public int[] getCounterDiagonal(int[][] matrix) {
-        return null;
+        int[] countDiagElts = new int[matrix.length];
+        for (int i = 0; i < matrix.length; i++) {
+            countDiagElts[i] = matrix[i][matrix.length - (i + 1)];
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/MatrixOperation.java]:
```suggestion
            counterDiagonalElts[i] = matrix[i][matrix.length - i + 1];
```
use full words at naming. it is much more understandable

**Review by Tarabarnik — APPROVED** [review]

File: src/main/java/core/basesyntax/MatrixOperation.java (hunk 2 above)
Comment (nevskyy) [reply src/main/java/core/basesyntax/MatrixOperation.java]: Pasha god morning. Comments deleted fm code.

**Comment by nevskyy** [comment]
Done

File: src/main/java/core/basesyntax/MatrixOperation.java (hunk 1 above)
Comment (Tarabarnik) [reply src/main/java/core/basesyntax/MatrixOperation.java]: this one is here yet

**Comment by nevskyy** [comment]
Completed

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**